packages/ui/src/server/controllers/availability.ts

```diff
@@ -53,7 +53,13 @@ function getAvailability(req: Request, clusters: Record<string, {id: string}>, o
 }
 
 export async function getClustersAvailability(req: Request, res: Response) {
-    const odinPath = getApp().config?.odinBaseUrl;
+    const {ytAuthCluster: cluster} = req.params;
+    const odinPath = _.get(
+        getApp().config,
+        `odinBaseUrl.${cluster}`,
+        getApp().config.odinBaseUrl,
+    ) as string;
```

Comment:
Please replace `getApp().config` with `req.ctx.config`, `getApp` was brought to break some cyclic dependencies in our code, it is not recommended to use it.
Also it is better to avoid usages of `_.get/_.set` cause they always break ts-typings:
```ts
const {odinBaseUrl} =  req.ctx.config;
const odinPath = 'string' === odinBaseUrl ? odinBaseUrl : odinBaseUrl?.[cluster];
```